android/src/main/java/tv/vizbee/rnsender/VizbeeCastButtonViewManager.java

```diff
@@ -21,4 +24,10 @@ protected VizbeeCastButtonView createViewInstance(ThemedReactContext reactContex
     public void onDropViewInstance(VizbeeCastButtonView buttonView) {
         super.onDropViewInstance(buttonView);
     }
+
+    @ReactProp(name = "color")
```

Comment:
Is it appropriate to say `tintColor` rather than color?
I was referring to the RN cheat sheet - https://github.com/vhpoet/react-native-styling-cheat-sheet#image and UIButton - https://developer.apple.com/documentation/uikit/uibutton?language=objc, based on these, it's better to name the property and the corresponding method as `tintColor`/`setTintColor`.